src/store/user/UserStateAtoms.ts

```diff
@@ -1,12 +1,17 @@
 import { atom } from "recoil";
 
-type UserData = {
-  id: string;
-  displayName: string;
-  imageUrl: string | null;
-};
+type UserData =
+  | {
+      id: string;
+      displayName: string;
+      imageUrl: string | null;
+      isLoggedIn: true;
+    }
+  | {
+      isLoggedIn: false | null;
+    };
 
```

Comment:
Refactor user state,
So now in the app you need to check first user is logged in to know you have user data